src/node/internal/internal_dns.ts

```diff
@@ -0,0 +1,94 @@
+import { sendDnsRequest } from 'node-internal:internal_dns_client';
+import { ok as assert } from 'node-internal:internal_assert';
+
+import { validateString } from 'node-internal:validators';
+
+export function getServers(): string[] {
+  throw new Error('Not implemented');
+}
+
+export function lookup(): void {
+  throw new Error('Not implemented');
+}
+
+export function lookupService(): void {
+  throw new Error('Not implemented');
+}
+
+export function resolve(): void {
+  throw new Error('Not implemented');
+}
+
+export function resolve4(): void {
+  throw new Error('Not implemented');
+}
+
+export function resolve6(): void {
+  throw new Error('Not implemented');
+}
+
+export function resolveAny(): void {
+  throw new Error('Not implemented');
+}
+
+export function resolveCname(): void {
+  throw new Error('Not implemented');
+}
+
+export function resolveCaa(): void {
+  throw new Error('Not implemented');
+}
+
+export function resolveMx(): void {
+  throw new Error('Not implemented');
+}
+
+export function resolveNaptr(): void {
+  throw new Error('Not implemented');
+}
+
+export function resolveNs(): void {
+  throw new Error('Not implemented');
+}
+
+export function resolvePtr(): void {
+  throw new Error('Not implemented');
+}
+
+export function resolveSoa(): void {
+  throw new Error('Not implemented');
+}
+
+export function resolveSrv(): void {
+  throw new Error('Not implemented');
+}
+
+export async function resolveTxt(hostname: string): Promise<string[][]> {
+  validateString(hostname, 'hostname');
+  const json = await sendDnsRequest(hostname, 'TXT');
+  assert(
+    json.Answer,
+    `DNS request should contain an "Answer" attribute but it didn't.`
+  );
+  // Each entry has quotation marks as a prefix and suffix.
+  // Node.js APIs doesn't have them.
+  return json.Answer.map((value) => [value.data.replaceAll('"', '')]);
```

Comment:
would value.data.slice(1, -1) be faster ?

Reply:
Nice catch. Yes, definitely!